source/features/clean-conversation-headers.tsx

```diff
@@ -41,7 +41,7 @@ async function initPR(): Promise<void> {
 	const baseBranch = base.title.split(':')[1];
 
 	// Replace the word "from" with an arrow
-	headBranch.previousSibling!.replaceWith(' ', <ArrowLeftIcon/>, ' ');
+	headBranch.previousSibling!.replaceWith(' ', <ArrowLeftIcon className="v-align-middle"/>, ' ');
```

Comment:
hmmm it doesn't seem to change anything here in Safari

Reply:
![Video_2021-04-14_230735](https://user-images.githubusercontent.com/16872793/114808496-615b3080-9d76-11eb-9b0b-6da9ed3d20e9.gif)

Reply:
Yeah, with or without this class that’s what it looks like

Reply:
Perhaps I didn’t notice. 👍 🚀